static/scss/payment.scss

```diff
@@ -20,10 +24,97 @@
     flex-direction: column;
     max-width: 285px;
     margin: 0 auto;
-    padding: 25px 0 75px 0;
+    padding-top: 25px;
 
     input {
       margin-bottom: 25px;
     }
   }
+
+  .payment-history-section {
+    background-color: $light-gray;
+    text-align: left;
+
+    .payment-history-block {
+      max-width: 850px;
+      margin: 0 auto;
+      padding: 60px 10px;
+    }
+
+    h3 {
+      padding: 0 0 45px 0;
+      margin: 0;
+      color: #000;
+      font-family: $sans-serif-font;
+      font-size: 15px;
+      text-transform: uppercase;
+      letter-spacing: 1px;
+
+      &::after {
+        content: "";
+        display: inline-block;
+        background-color: $med-gray;
+        margin-left: 15px;
+        height: 1px;
+        width: 70px;
+        position: relative;
+        bottom: 5px;
+      }
+    }
+
+    table {
```

Comment:
it looks in the JSX above like the `table` and `h3` elements which are being styled here are nested within the `.payment-history-block` selector - any reason why the blocks for these selectors aren't inside the block for `.payment-history-block` here in the CSS?